apps/web/src/app/api/documentLogs/[id]/route.ts

```diff
@@ -14,28 +12,24 @@ export const GET = errorHandler(
         params,
         workspace,
       }: {
-        params: { id: number }
+        params: { id: string }
         workspace: Workspace
       },
     ) => {
       const id = params.id
+      const result = await fetchDocumentLogWithMetadata({
+        workspaceId: workspace.id,
+        documentLogId: +id,
```

Comment:
i'm getting confused, is this ID or uuid? if it's a uuid why is the prop called documentlogid and we are coercing to number? if it's number... then the i don't understand why you are doing these changes?

Reply:
Is both. We have a little mess. In the UI we use `uuid` in the URL. But in the evaluationResult.documentLogId. So when we request the logs we need both apis. The DRY here is that `fetchDocumentLogWithMetadata` handle both cases